Rocket.Chat/Extensions/UIWindowExtensions.swift

```diff
@@ -41,9 +41,13 @@ extension UIWindow {
         }
 
         /// The presenting view controllers view doesn't get removed from the window as its currently transistioning and presenting a view controller
```

Comment:
How does this issue materialize in our app? Like what are the steps to reproduce it?

I was under the impression that this problem only exist if the dismiss method is not called on the parent vc.